docs/modules/data-structures/pages/working-with-external-data.adoc

```diff
@@ -31,15 +31,88 @@ NOTE: The data store that you choose must be a centralized system that is access
 
 NOTE: The `MapStore` interface extends the `MapLoader` interface. Therefore, all methods and configuration parameters of the `MapLoader` interface are also available on the `MapStore` interface.
 
+== Defining External Data Stores (Beta)
```

Comment:
```suggestion
== Configuring Connections to External Data Stores (Beta)
```

Can this be any data store or does it have to allow JDBC connections?

Reply:
Yes, it can be any data store we come up with, currently we've implemented on JDBC-based